ports/voidstranger/Void Stranger.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+export PORT_32BIT="N"
+GAMEDIR="/$directory/ports/voidstranger"
+export LD_LIBRARY_PATH="/usr/lib:/usr/lib32:/$GAMEDIR/lib:$LD_LIBRARY_PATH"
+
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+$ESUDO chmod 666 /dev/tty0
+
+cd $GAMEDIR
+
+export GMLOADER_DEPTH_DISABLE=1
+export GMLOADER_SAVEDIR="$GAMEDIR"
+export GMLOADER_PLATFORM="os_linux"
+
+# log the execution of the script into log.txt
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+# here is what we are expecting to get out of the patching process
+expected_chksm111="f38006605356a868087ccff137eb274d"
+
+#first time setup check
+first_time=false
```

Comment:
Instead of lengthy checksum scans every run, you can do it in a function. Then, do a conditional to test if "installed" exists, if not, run the function. An example of this method is in Undertale Yellow.

Reply:
I'll take a look!